operators/distiller-pipeline/distiller_pipeline/__init__.py

```diff
@@ -0,0 +1,981 @@
+import multiprocessing
```

Comment:
Just question about the naming, so this operator encapsulates the current streaming/counting distiller pipeline? I wonder if distiller-streaming might be a better name? The idea of have a pipeline inside an operator is a little confusing to me.

Reply:
yes we can rename.